cmd2/argcomplete_bridge.py

```diff
@@ -6,11 +6,16 @@
     import argcomplete
 except ImportError:  # pragma: no cover
     # not installed, skip the rest of the file
-    pass
-
+    DEFAULT_COMPLETER = None
```

Comment:
@anselor Would you please review the changes I made to your PR.  These changes were to support older versions of ``argcomplete`` which have a few API differences including:
- ``FilesCompleter`` is not accessible at the top level
- Fewer keyword arguments to an ``__init__`` method

These older versions of ``argcomplete`` would commonly be found on older versions, in particular Python 3.4.

Reply:
Looks good to me.